open-apparel-registry/open-apparel-registry
# Update header, footer, and font

## Overview

The info website has been updated with a new header, footer, and font. In
order to keep users' experiences as similar as possible between
the info site and the map site, updates the header, footer, and font to match.

A code sample was provided for the info website's header and footer,
and wireframes were provided for the full-width view of the map site's
header and of the info site's mobile header. As much consistency as
possible with these wireframes has been used. Differences include
adding authentication, translation, and legal links to the mobile
navigation, and adding the API documentation link to the full-width
footer.

The font has been updated to the new font.

Several components which are no longer needed following the redesign have been removed. 

Connects #1489, #1471

## Demo
Mobile
<img width="496" alt="Screen Shot 2021-10-20 at 11 52 07 AM" src="https://user-images.githubusercontent.com/21046714/138127604-c7a7bfca-c536-439d-9183-e02296b6cf93.png">
![Oct-20-2021 11-40-58](https://user-images.githubusercontent.com/21046714/138126505-5749e891-33a1-4862-a1cc-7a2db6ffb2f3.gif)

Web
<img width="1394" alt="Screen Shot 2021-10-20 at 11 38 35 AM" src="https://user-images.githubusercontent.com/21046714/138127119-9ea71d47-87e5-400d-82c3-39671de85668.png">
<img width="1395" alt="Screen Shot 2021-10-20 at 11 39 15 AM" src="https://user-images.githubusercontent.com/21046714/138127211-ddf7371e-b522-41ba-9eac-87fe90dcbe6b.png">
<img width="1272" alt="Screen Shot 2021-10-20 at 11 50 17 AM" src="https://user-images.githubusercontent.com/21046714/138127328-605ae7ff-6660-48d1-bb52-ab72a6efbec5.png">
![Oct-20-2021 11-47-40](https://user-images.githubusercontent.com/21046714/138126865-cbfbbceb-991f-4c5c-9bf0-bdfa32f192a6.gif)

Google Translate
<img width="1391" alt="Screen Shot 2021-10-20 at 11 38 47 AM" src="https://user-images.githubusercontent.com/21046714/138127054-c04c24f7-9889-4689-b4e3-fc22127cfab9.png">
![Oct-20-2021 11-46-53](https://user-images.githubusercontent.com/21046714/138126714-9229820d-124f-4096-aa92-cdc54a78c2c0.gif)

## Notes

[Mobile Design](https://marvelapp.com/prototype/76h8ibe/screen/81914627)
[Web Design](https://marvelapp.com/prototype/76h8ibe/screen/81914627)
[Codepen](https://codepen.io/niceandserious/pen/85042a94117e253b14acc0bf92ec02e9)

The wireframes didn't include the API documentation link currently in the header. Per discussion with Scott, we moved this link to the footer on web view and removed it in mobile view, under the assumption that generally API users will be using a desktop/laptop. 

The mobile menu wireframes didn't include a mockup of the changes required for the differences between the info site and the map site. After discussion with Scott, I used similarly styling to the same links in the header for contribute and login links. I added the authentication menu as a separate menu, similarly to how we had it previously. 

The cookie preferences, terms of service, and privacy policy links/buttons are in the footer on web, but we hide the footer on mobile. We added these links as a submenu called 'more' but should discuss this label with the OAR team (changes to the title of the submenu can be held off for future updates).

## Testing Instructions

* Run `./scripts/server` and navigate to the website. 
* With a wide screen, click through the dropdown links. Login and test the auth menu. 
* Check the footer links appear as expected. 
* Test the Google translate dropdown. Click several languages and confirm the button label updates. Click back to English and confirm the label says 'EN'.
* With a narrow screen, click through the hamburger menu and the auth menu. Logout and click through the hamburger menu. 
* Setup a user account with embed config privileges and view the embed preview for that user with a noticeably different font selected. Confirm that the embedded map renders as expected with the appropriate font. 

## Checklist
